src/main/kotlin/org/wycliffeassociates/otter/jvm/device/audioplugin/AudioPlugin.kt

```diff
@@ -0,0 +1,26 @@
+package org.wycliffeassociates.otter.jvm.device.audioplugin
+
+import io.reactivex.Completable
+import io.reactivex.schedulers.Schedulers
+import org.wycliffeassociates.otter.common.data.audioplugin.AudioPluginData
+import org.wycliffeassociates.otter.common.data.audioplugin.IAudioPlugin
+import java.io.File
+
+class AudioPlugin(val pluginData: AudioPluginData) : IAudioPlugin {
+    override fun launch(file: File): Completable {
+        return Completable
+                .fromCallable {
+                    // Build and start the process
+                    val process = ProcessBuilder(
+                            listOf(
+                                    pluginData.executable,
+                                    *(pluginData.args.toTypedArray()),
+                                    file.toString()
+                            )
+                    )
+                            .start()
```

Comment:
styling

Reply:
I guess the intent here was to tab off the ProcessBuilder, but it isn't clear when you look down vertically (especially since it lines up with the list of)

The specific guideline is ambiguous ("The first call in the chain usually should have a line break before it, but it's OK to omit it if the code makes more sense that way.") I think that having the .start on the end of the close parentheses is probably appropriate here